contracts/utils/Ownable.sol

```diff
@@ -26,11 +30,17 @@ abstract contract Ownable {
         nominatedOwner = address(0);
     }
 
+    /// @notice Give up the ownership of a contract
```

Comment:
```suggestion
    /// @notice Give up the ownership of this contract
```

Reply:
The `Ownable` is an abstract contract so that's why. I replaced by `Give up the ownership` to keep it simple.

Reply:
Adjusted in 7a91187197c37a8285b47ee4f3227bad434ebccb.